src/components/pages/aboutTD/AboutTD.tsx

```diff
@@ -0,0 +1,211 @@
+      <p>{position}</p>
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          textAlign: 'center',
+          justifyContent: 'center',
+        }}>
+        <p> {stringRepr} </p>
+      </div>
+      {mail.map((m, i) => (
+        <a
+          href={`mailto:${mail}`}
+          key={i}
+          style={{ marginBottom: i === mail.length - 1 ? '.5rem' : '0rem' }}>
+          {m}
+        </a>
+      ))}
+    </div>
+  );
+};
+
+interface ICommitte {
+  name:
+    | 'Arrangementkomite'
+    | 'Bedriftskomite'
+    | 'Nettsidekomite'
+    | 'TD på tapp'
+    | 'UiT Hack';
+}
+
+const Committe: React.FC<ICommitte> = ({ name, children }) => {
+  // committee description is passed as children
+  return (
+    <div className="committee">
+      <h4> {name} </h4>
+      {children}
+    </div>
+  );
+};
+
+const TDcommittees: React.FC = () => {
+  return (
+    <div className="committeeWrapper">
+      <Committe name="Arrangementkomite">
+        <p>
+          {' '}
+          Arrangementkomiten jobber med å lage forskjellige arrangement alt fra
+          fester og skiturer til musikkbingo. Er dette noe som høres spennende
+          ut ta kontakt med oss på{' '}
+          <a href="mailto:arrangementsansvarlig@td-uit.no">mail</a>
+        </p>
+      </Committe>
+
+      <Committe name="Bedriftskomite">
+        <p>
+          Bedriftskomiteen jobber med å komme i kontakt og lage alle
+          bedriftsarrangementer som gjøres i regi av TD. Lyst til å høre mer om
+          dette ta kontakt med{' '}
+          <a href="mailto:bedriftskommunikasjon@td-uit.no">
+            bedriftskommunikasjons ansvarlig
+          </a>
+        </p>
+      </Committe>
+      <Committe name="Nettsidekomite">
+        <p>
+          {' '}
+          Lyst til å lære frontend og backend utvikling eller bare har lyst til
+          å legge til noen features. Ta{' '}
+          <a href="mailto:nettside-ansvarlig@td-uit.no">kontakt</a> eller sjekk
+          ut githuben vår for{' '}
+          <a href="https://github.com/td-org-uit-no/tdctl-frontend">frontend</a>{' '}
+          og <a href="https://github.com/td-org-uit-no/tdctl-api">backend </a>
+        </p>
+      </Committe>
+      <Committe name="TD på tapp">
+        <p>
+          {' '}
+          TD på tapp er TD sin ølbryggings komite. Her brygges det øl til
+          forskjellige anledninger. Er du interessert i ølbrygging eller har
+          lyst til å bli med? send oss en{' '}
+          <a href="mailto:studentkommunikasjon@td-uit.no">mail</a>
+        </p>
+      </Committe>
+      <Committe name="UiT Hack">
+        <p>
+          TD arrangere årlig sin egen CTF kalt{' '}
+          <a href="https://uithack.no/">UiTHack</a> og vi trenger folk til å
+          være med på å lage forskjellige oppgaver. Er du interissert i hacking
+          og CTF ta kontakt med{' '}
+          <a href="mailto:studentkommunikasjon@td-uit.no">teknisk</a>
```

Comment:
Should the descriptions maybe be gotten from a file so that it is easier to change retrospectively?